gofish/2025/google-login.html

```diff
@@ -0,0 +1,218 @@
+      margin-bottom: 12px;
+    }
+
+    .input input {
+      width: 100%;
+      padding: 12px 10px;
+      font-size: 15px;
+      border-radius: 6px;
+      border: 1px solid #cfcfcf;
+      background: #fff;
+      outline: none;
+    }
+
+    .input input:focus {
+      box-shadow: 0 0 0 3px rgba(26,115,232,0.1);
+      border-color: #c1d7ff;
+    }
+
+    .small-link {
+      display: block;
+      margin-top: 6px;
+      margin-bottom: 12px;
+      color: #1a73e8;
+      font-weight: 600;
+      text-decoration: none;
+      font-size: 13.5px;
+    }
+
+    .info {
+      color: var(--muted);
+      font-size: 13px;
+      margin: 12px 0;
+      line-height: 1.45;
+    }
+
+    .bottom-row {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      margin-top: 6px;
+    }
+
+    .create-link {
+      color: #1a73e8;
+      font-weight: 600;
+      text-decoration: none;
+      font-size: 13.5px;
+    }
+
+    .next-btn {
+      background: var(--primary);
+      color: #fff;
+      border: none;
+      padding: 10px 20px;
+      border-radius: 22px;
+      font-weight: 700;
+      font-size: 14px;
+      cursor: pointer;
+      box-shadow: 0 4px 10px rgba(26,115,232,0.16);
+    }
+
+    .next-btn:active { transform: translateY(1px); }
+
+    .footer-area {
+      position: absolute;
+      bottom: 16px;
+      left: 0;
+      width: 100%;
+      display: flex;
+      justify-content: center;
+      gap: 20px;
+      font-size: 13px;
+      color: var(--muted);
+      flex-wrap: wrap;
+    }
+
+    .footer-links a {
+      color: var(--muted);
+      text-decoration: none;
+    }
+
+    @media (max-width: 420px) {
+      .card {
+        width: 100%;
+        padding: 20px 22px 28px 22px;
+        border-radius: 16px;
+      }
+      h1 { font-size: 26px; }
+    }
+  </style>
+</head>
+<body>
+  <div class="card">
+    
+    <!-- Placeholder logo area -->
+    <div class="logo">
+      <img src="../../download (1).png" alt="Google Logo Placeholder">
+    </div>
+
+    <h1>Sign in</h1>
+    <div class="sub">Use your Google Account</div>
+
+    <form id="emailForm" novalidate>
+      <label class="input">
+        <input type="email" id="email" placeholder="Email or phone" required />
+      </label>
+
+      <a href="#" class="small-link">Forgot email?</a>
+
+      <div class="bottom-row">
+        <a href="#" class="create-link">Create account</a>
+        <a href="password.html" class="next-btn" style="text-decoration:none;display:inline-block;text-align:center;">Next</a>
+      </div>
+    </form>
+  </div>
+
+  <div class="footer-area">
+    <span>English (United States)</span>
+    <div class="footer-links">
+      <a href="#">Help</a>
+      <a href="#">Privacy</a>
+      <a href="#">Terms</a>
+    </div>
+  </div>
+
+  <script>
+    document.getElementById('emailForm').addEventListener('submit', e => {
+      e.preventDefault();
+      alert('Demo page only — no real login occurs.');
+    });
```

Comment:
You should make this redirect to your `phew.html`, since it would classify as a breach.